website/content/en/blog/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2022-10-10
+linkTitle: "Kubeflow on AWS Blog"
+description: "Welcome to the **blog** section of Kubeflow on AWS!"
+type: "docs"
+layout: "single"
+---
+
+In this section, we share posts, articles, and videos illustrating how some of our customers have leveraged **AWS distribution of Kubeflow** to optimize large-scale and compute-intensive machine learning workloads, we provide ML infrastructure recommendations and best practices, and we inform you on Kubelow on AWS coming training sessions and available recordings.
+
+Want to **try the latest version of Kubeflow on AWS**? Follow the steps in [Kubeflow on AWS documentation](http://localhost:1313/kubeflow-manifests/main/docs/deployment/prerequisites/).
+
+[**Read**](#read) from our curated list of posts or [**Watch**](#watch) a recorded session.
+
+## Read
+
+A curated list of blogs illustrating how to build flexible, scalable ML workflows on Kubernetes using AWS distribution of Kubeflow. 
+Find case studies, use cases, best practices, benchmarks, and more.
+
+1. **[ Architecture ]** 
+    [Build and deploy a scalable machine learning system on Kubernetes with Kubeflow on AWS](https://aws.amazon.com/blogs/machine-learning/build-and-deploy-a-scalable-machine-learning-system-on-kubernetes-with-kubeflow-on-aws/). Based on Kubeflow-1.4.
```

Comment:
This blog post is based on Kubeflow-1.4 and is listed here for educational purpose. AWS distribution of Kubeflow might have changed since then and we recommend you to install the latest [version](https://awslabs.github.io/kubeflow-manifests/docs/).